addons/MacroPolo/README.md

```diff
@@ -0,0 +1,11 @@
+MacroPolo is a macro management addon that enables the use of a 'back' command thus allowing shared sets across different macro books.
+
+Usage within macros: 
+/macropolo [current_book]-[current_set] [destination_book] [destination_set]
+/console macropolo [current_book]-[current_set] [destination_book] [destination_set]
+
+Example: '/macropolo 3-1 1-5': records the current macro position, as Book 3 Set 1 and then sends the user to Book 1, Set 5.
+
+You may then use either of these commands in a macro to return to the previous book and set.
+/macropolo back
+/console macropolo back
```

Comment:
Nothing to critique here, but I'd beef up the file with some markdown. That's why it's called `.md` after all :)